### PR TITLE
Release notes for Feb 2022

### PR DESCRIPTION
**Pull request by HBehrens**

### CODE REVIEW

File: docs/mcu/nrf-connect-sdk-guide.mdx [hunk 1]
```diff
@@ -13,13 +13,14 @@ import TabItem from "@theme/TabItem";
...
+releases ranging from v1.4.0 up to v1.6.0. All versions of the nRF Connect SDK
 since v1.6.0 include Memfault when downloaded from Nordic's website.
 
 :::tip
 
-nRF Connect SDK v1.6.0, and all versions since, has built-in support for the Memfault Firmware SDK on
-nRF9160-based targets. Nordic Semiconductor has released some
+nRF Connect SDK v1.6.0, and all versions since, has built-in support for the
+Memfault Firmware SDK on nRF9160-based targets. Nordic Semiconductor has
+released some
```
Comment (HBehrens) [inline docs/mcu/nrf-connect-sdk-guide.mdx:23]: left-over from #184, see https://github.com/memfault/memfault-docs/pull/184#issuecomment-1060452559

File: blog/2022-02-28.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+title: February 2022
+---
+
+## Android SDK Bort 4.0
+
+Memfault’s AOSP SDK “Bort”received a major
```
Comment (martijnthe) [inline blog/2022-02-28.mdx:7]:
```suggestion
Memfault’s AOSP SDK “Bort” received a major
```

File: blog/2022-02-28.mdx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+[update 4.0 with many improvements](https://github.com/memfault/bort/blob/master/CHANGELOG.md#v400---february-09-2021)
+such as additional support for Android 12 and multi-user compatibility.
+
+We added [support for Custom Metrics](/docs/android/android-custom-metrics) so
+that devices can report product-specific numerical values, strings, and state
```
Comment (martijnthe) [inline blog/2022-02-28.mdx:12]:
```suggestion
that devices can report product-specific numerical values, strings and state
```

File: blog/2022-02-28.mdx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+[Time-Series Metrics and Device Attributes](/docs/platform/metrics).
+
+![Visualized Metrics per Device on its Timeline](/img/blog/2022-02-28-deviceTimeline.png)
+
+Another rather advanced feature enables devices without direct Internet
+connection to report crashes and metrics to Memfault. Data is packaged as `.mar`
+(Memfault Archive) files and vendors may upload them at a later time or upon
+request (e.g. downloaded periodically via USB or in a local network by an
+auxiliary device). This allows vendors to use Memfault in scenarios with
+advanced security requirements.
```
Comment (martijnthe) [inline blog/2022-02-28.mdx:28]:
```suggestion
strict security requirements.
```

File: blog/2022-02-28.mdx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+## Dashboard changes to charts
+
+[Memfault’s Dashboard](/docs/platform/charts#overview-dashboard) provides an
+overview of your fleet at a glance. We have updated the charts “Active Devices”
+(more sources used as signal) and “Software Versions” (only active devices
+considered) to better compare apples to apples.
```
Comment (martijnthe) [inline blog/2022-02-28.mdx:35]:
```suggestion
(only active devices considered) and “Software Versions” (more sources
used as signal) to better compare apples to apples.
```

File: blog/2022-02-28.mdx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+![Dashboard showing Acive Devices and Version distribution](/img/blog/2022-02-28-dashboard.png)
+
+The visible time range for “Software Versions” can now be changed from “2
+months” all the way down to just “24 hours”. Since the same chart is now being
+used on the [Cohort details page](/docs/platform/ota#cohorts-view), it not only
+allows you to see long-term trends, but also acts as timely signal to observe
+the effect of an ongoing OTA software rollout.
+
+The charts “Seen Devices” and “Received Events” had been removed.
```
Comment (martijnthe) [inline blog/2022-02-28.mdx:45]:
```suggestion
The charts “Seen Devices” and “Received Events” have been removed.
```

(Rest is also using present perfect tense?)

**Review by martijnthe — APPROVED** [review]

**Comment by HBehrens** [comment]
Thank you for proof-reading, @martijnthe – will adjust and repopulate change back to Notion… (this workflow stinks)

File: blog/2022-02-28.mdx (hunk 3 above)
Comment (HBehrens) [reply blog/2022-02-28.mdx:12]: Nope. The author is an advocate of the [Oxford comma ](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it) ;)

**Pull request merged**